src/main/java/frc/robot/Robot.java

```diff
@@ -131,6 +134,9 @@ public void disabledInit() {
       m_robotContainer.robotDrive.setIdleMode(IdleMode.kCoast);
       m_robotContainer.climber.coastWinch();
     }
+    if (m_autonomousCommand != null) {
+      m_robotContainer.robotDrive.tankDriveVolts(0, 0);
+    }
```

Comment:
This block ought to be near the top of disabledInit, so it happens immediately. You need to set m_autonomousCommand to null so it does not keep doing this.

Reply:
Concur